src/components/appointmentItem/AppointmentOption.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {StyleSheet, View, Text} from 'react-native';
+
+import {spacing, theme} from '../../theme/index';
+
+type AppointmentOptionProps = {
+  text?: string;
+  day?: string;
+};
```

Comment:
could you please tell me why optional here
move this one AppointmentOptionProps to model and import

Reply:
optional because we have 3 types of option (look at design)